### ISSUE TITLE
MacOS: binary not working

### ISSUE THREAD

**Issue by ngaloppo** [naftulikay/aws-env#26]
```sh
aws-env list
[2023-02-15T14:41:22.524835-08:00 ERROR aws_env::loader (main)] Unable to read directory: No such file or directory (os error 2)
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/main.rs:119:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by ngaloppo** [comment]
At the very least, a more intelligible error message would be nice :)

**Comment by naftulikay** [comment]
@ngaloppo can you run it with `RUST_BACKTRACE=1`? The file/line doesn't exactly tell me much: [src/main.rs:119](https://github.com/naftulikay/aws-env/blob/master/src/main.rs#L119). 

I suspect that the `dirs` crate might be failing for some reason on macOS. Apologies, I'm on Linux and I don't have a Mac to test. I can probably setup test runs in GitHub Actions for macOS.

**Comment by ngaloppo** [comment]
Sure, here you go. 

```
[2023-02-16T10:19:14.370899-08:00 ERROR aws_env::loader (main)] Unable to read directory: No such file or directory (os error 2)
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/main.rs:119:10
stack backtrace:
   0:        0x10293a1b1 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h4edbddd26fc45697
   1:        0x1029537db - core::fmt::write::h266c957f12b1655a
   2:        0x1029364ca - std::io::Write::write_fmt::h65d4ee621e8d810d
   3:        0x10293b995 - std::panicking::default_hook::{{closure}}::h5b4886cd9f93dfca
   4:        0x10293b57f - std::panicking::default_hook::h986d152c7bd7e732
   5:        0x10293c080 - std::panicking::rust_panic_with_hook::h151f3b3bf37b4f17
   6:        0x10293bb1e - std::panicking::begin_panic_handler::{{closure}}::hf6660086d9ebd48c
   7:        0x10293a627 - std::sys_common::backtrace::__rust_end_short_backtrace::h34152178ea368a9a
   8:        0x10293ba8a - _rust_begin_unwind
   9:        0x102965a1f - core::panicking::panic_fmt::hb64a2db862b4aca0
  10:        0x102965a95 - core::result::unwrap_failed::hd3218c9fc9801751
  11:        0x10285fe81 - aws_env::main::h54e885e594711ebc
  12:        0x102847b46 - std::sys_common::backtrace::__rust_begin_short_backtrace::h9283ba691d742f05
  13:        0x102852b8c - std::rt::lang_start::{{closure}}::hce3966771cae55b7
  14:        0x102939e9f - std::rt::lang_start_internal::h59ad95b3bebc83b2
  15:        0x1028620a9 - _main
  16:     0x7ff812aaa310 - <unknown>
```

**Comment by naftulikay** [comment]
@ngaloppo this appears to be fixed in #27, please test and reopen if still broken. I'm going to create a new release for this, `v2.0.1`.
